### PR TITLE
RD-6652 - Update for RHEL 8

### PR DESCRIPTION
**Pull request by jschmidt-civicactions**
Split setup.sh into two versions to support both RHEL 7 / CentOS 7 and RHEL 8.
Added new tailoring for GlobalNET cloud instances using RHEL 8.
Updated documentation.

### CODE REVIEW

File: packer-rhel8-ami.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "{{user `region`}}",
+    "encrypt_boot": false,
+    "ena_support": true,
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "name": "RHEL-8.4.0_HVM-*x86_64*",
```
Comment (sfmcgee) [inline packer-rhel8-ami.json:12]: Don't hate me, but it looks like RHEL 8.5 GA came out in November (after Dan did this work). What do you think about trying it out?

Comment (jschmidt-civicactions) [reply packer-rhel8-ami.json:12]: Sure, why not.

File: ssg-rhel8-ds-tailoring.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_mount_option_var_log_nosuid" selected="false"/>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_mount_option_var_tmp_nodev" selected="false"/>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_mount_option_var_tmp_noexec" selected="false"/>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_mount_option_var_tmp_nosuid" selected="false"/>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_sssd_enable_certmap" selected="false"/>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_usbguard_generate_policy" selected="false"/>
+    <!-- RD-6613 - New test requires explicit set of allowed local users -->
+    <xccdf:set-value idref="xccdf_org.ssgproject.content_value_var_accounts_authorized_local_users_regex">^(adm|bin|chrony|daemon|dbus|ec2-user|ftp|games|halt|lp|mail|nobody|operator|polkitd|shutdown|sshd|sssd|sync|systemd-coredump|systemd-resolve|tss|unbound|{{ account_users|join("|",attribute='name')}}|{{ developer_users|join("|",attribute='name')}}|{{ system_users|join("|",attribute='name')}}|{{ system_service|join("|",attribute='name')}})$</xccdf:set-value>
```
Comment (sfmcgee) [inline ssg-rhel8-ds-tailoring.xml:61]: These are unique variables from the GlobalNET Infrastructure and should be removed: 
{{ account_users|join("|",attribute='name')}}|{{ developer_users|join("|",attribute='name')}}|{{ system_users|join("|",attribute='name')}}|{{ system_service|join("|",attribute='name')}}

Comment (jschmidt-civicactions) [reply ssg-rhel8-ds-tailoring.xml:61]: Roger that

File: ssg-rhel8-ds-justifications.yaml [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+  - rule_id: xccdf_org.ssgproject.content_rule_usbguard_generate_policy
+    severity: medium
+    - justification: |
+        CloudFormation instances do not have USB devices and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_value_var_accounts_authorized_local_users_regex
+    severity: medium
+    - justification: |
+        Authorized user list customized based on GlobalNET base configuration.
```
Comment (sfmcgee) [inline ssg-rhel8-ds-justifications.yaml:279]: Sorry, didn't catch this before - we should remove GlobalNET from the Hailstone justification. It's just the default local user list that we're customizing for.

**Comment by sfmcgee** [comment]
Looks good, thanks for all the updates!

**Review by sfmcgee — APPROVED** [review]

**Pull request merged**